PluginCode/src/main/java/smr/cs/ualberta/libcomp/action/ReplacementAction.java

```diff
@@ -595,20 +612,20 @@ public void replaceRequestedMaven(@NotNull final AnActionEvent event) throws Par
         int locationStartOfImport;
         int locationEndOfImport;
 
-        while (currentLine < MavenListObjects.size()) {
-            if (MavenListObjects.get(currentLine).getImportLocation() == clickedLineNumber) {
-                from_library = MavenListObjects.get(currentLine).getImportLib();
-                locationStartOfImport = MavenListObjects.get(currentLine).getFromlocation();
-                locationEndOfImport = MavenListObjects.get(currentLine).getTolocation();
-                int finalFrom_library = from_library;
+        while (currentLine < listObjects.size()) {
```

Comment:
so what is listObjects?

Reply:
it's a general name for ArrayList<DependencyStatement>. a list of maven or gradle object

Reply:
can we use a more description name then like `dependencyList`?

Reply:
Sounds good.